castle/cms/static/patterns/survey.js

```diff
@@ -113,7 +113,8 @@ define([
       try {
         xhr.send(data);
         xhr.onload = function() {
-          self.renderInvite(self.survey_tile, self.survey_data, xhr.response);
+          self.invite_data = JSON.parse(xhr.response);
```

Comment:
xhr.response is only not an object for IE? Can we be sure this doesn't break non-IE?

Maybe even wrap in try/catch?